health-probe-proxy/example.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: health-probe-proxy
+  namespace: kube-system
+  labels:
+    component: health-probe-proxy
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
remove this line?

Reply:
This is useful when using addon manager to deploy health probe proxy.